SingularityUI/app/templates/deployDetail/activeTasks.hbs

```diff
@@ -0,0 +1,76 @@
+<div class="page-header">
+    <h2>Active Tasks</h2>
+</div>
+{{#if data.length}}
+    <div class="table-container">
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>
+                        Name
+                    </th>
+                    <th class="hidden-sm hidden-xs">
+                        Last state
+                    </th>
+                    <th class="hidden-sm hidden-xs">
+                        Deploy
+                    </th>
+                    <th class="hidden-sm hidden-xs">
+                        Started
+                    </th>
+                    <th class="hidden-xs">
+                        Updated
+                    </th>
+                    <th class="hidden-xs">
+                        {{! Actions }}
```

Comment:
Let's make this header (the one above the `...` link) say `Logs` , the tooltip syas it, but a header would be helpful to know at first glance what it is